warehouse/templates/manage/account.html

```diff
@@ -300,6 +300,21 @@ <h2 class="sub-title">{% trans %}Account details{% endtrans %}</h2>
             Displayed on your <a href="{{ href }}">public profile</a>
           {% endtrans %}
         </p>
+        <label class="form-group__label" for="is_email_private">
+          {% trans %}️Keep my email address private{% endtrans %}
+          {% if save_account_form.name.flags.required %}
```

Comment:
This seems to be from the `name` field before this field.

Reply:
Yup, cut-n-paste mixup.  Thanks for the catch.